hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/AbstractKeyDeletingService.java

```diff
@@ -613,6 +615,10 @@ public AtomicLong getRunCount() {
     return runCount;
   }
 
+  public AtomicLong getCallId() {
+    return callId;
+  }
```

Comment:
Instead of exposing `callId`, please create a `protected` method for submitting request:

```java
  protected OzoneManagerProtocolProtos.OMResponse submitRequest(OMRequest omRequest) throws ServiceException {
    return OzoneManagerRatisUtils.submitRequest(ozoneManager, omRequest, clientId, callId.incrementAndGet());
  }
```

and replace all calls of `OzoneManagerRatisUtils.submitRequest` in `AbstractKeyDeletingService` and subclasses.